setup.py

```diff
@@ -59,7 +59,7 @@
     name='django-openid-auth',
     version=VERSION,
 
-    packages=find_packages(),
+    packages=find_packages(include=['django_openid_auth*']),
```

Comment:
Why do we need to keep using `find_packages` here instead of just listing the one package in the repo?

Reply:
Because the package list has to specify every package and subpackage.  The complete list would be: 
```
['django_openid_auth', 'django_openid_auth.management', 'django_openid_auth.migrations', 'django_openid_auth.south_migrations', 'django_openid_auth.management.commands']
```
I think `include` naming the top-level root of the package tree is a good compromise.

Reply:
Huh, I think I've been skipping enumeration of the sub-packages in many packages I've worked on, and they seem to work ok.  I can see that the docs (briefly and vaguely) recommend enumerating all the sub-packages, but I wonder what the difference is when you don't.